CLI/Import - add tests for import config-files

rplevka opened the pull request:
* fixed subcommand `config_file_with_tr_data`
    * added 2 cases to `tests.foreman.cli.test_import`
    * modified `get_sat6_id()` to accept entity key as an optional parameter

```bash
$ nosetests -m test_import_config_files_default test_import.py -vs
@test: Import all Config Files from the default data set ... ok

----------------------------------------------------------------------
Ran 1 test in 205.435s

OK

$ nosetests -m test_reimport_config_files_negative test_import.py -vs
@test: Repetitive Import of all Config Files from the default ... 

ok

----------------------------------------------------------------------
Ran 1 test in 178.847s

OK

```

elyezer commented:
This requires a rebase.

sthirugn commented on tests/foreman/cli/test_import.py:
```diff
@@ -1540,3 +1562,77 @@ def test_import_snippets_default(self):
+        (predefined source)
+        @feature: Import Config Files
+        @assert: All Config Files are imported
+        """
```
please format the docstring like other tests.

sthirugn commented on tests/foreman/cli/test_import.py:
```diff
@@ -1540,3 +1562,77 @@ def test_import_snippets_default(self):
+        data set (predefined source)
+        @feature: Repetitive Import Config Files
+        @assert: All Config Files are imported only once
+        """
```
same comment as above.

sthirugn commented:
ACK pending comments/rebase

rplevka commented:
rebased, comments addressed

sthirugn commented:
ACK

rplevka commented:
updated by changes introduced in PR #2873.
+ rebased

```
$ nosetests -m test_import_config_files_default test_import.py
.
----------------------------------------------------------------------
Ran 1 test in 100.779s

OK
$ nosetests -m test_reimport_config_files_negative test_import.py
.
----------------------------------------------------------------------
Ran 1 test in 124.611s

OK
```

lpramuk commented:
ACK

rplevka commented:
since the commit modifies a method `get_sat6_id()` which is being used by the other tests as well, here's the result from on of such tests to demonstrate, the change didn't change the behaviour:
```
$ nosetests -m test_import_host_collections_default test_import.py
.
----------------------------------------------------------------------
Ran 1 test in 39.333s

OK
```
